uktrade/data-hub-frontend
# Fix appearance of links inside the Panel

## Description of change

Fixes an accessibility issue when focused links inside the `Panel` component had a white text on yellow background.

## Test instructions

1. Go to `/companies/<id>/referrals/send`
2. Focus the _Read more guidance_ link in the _When to send a referral_ panel
3. The text color of the link should be the same as the panel's background color on yellow background

## Screenshots
### Before

<img width="705" alt="Screen Shot 2021-05-06 at 2 02 13 PM" src="https://user-images.githubusercontent.com/2333157/117302676-cdabea00-ae73-11eb-8bd1-dda2b8900697.png">

### After
<img width="705" alt="Screen Shot 2021-05-06 at 1 59 07 PM" src="https://user-images.githubusercontent.com/2333157/117302669-cc7abd00-ae73-11eb-84fa-c64395d309ba.png">
